tests/unit/test_winmanifest.py

```diff
@@ -548,7 +548,7 @@ def test_manifest_write_to_exe_non_ascii_characters(tmp_path):
     </security>
   </trustInfo>
 </assembly>
-""".encode("utf-8") # noqa: E501
+""".encode("utf-8") # noqa: E128,E261,E501
```

Comment:
I think E261 here is required only because the noqa comment itself is missing two whitespaces?

The other four manifest strings do not seem to require E261 at all.